### PR TITLE
Codemirror blob: enable dy default

### PR DESCRIPTION
**Pull request by taras-yemets**
Enables CodeMirror blob view by default.

## Test plan
CI passes

## App preview:

- [Web](https://sg-web-taras-yemets-enable-cm-blob-view.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by philipp-spiess** [comment]
@taras-yemets What's the status on this PR?

**Comment by taras-yemets** [comment]
> @taras-yemets What's the status on this PR?

Now the CodeMirror blob view is enabled for Sourcegraph org users on dotcom. We [are going to enable it by default](https://sourcegraph.slack.com/archives/C04931KQVRC/p1671443208572209) in the next release if there are no outstanding blockers.
I'll audit the existing issues list and update the PR status.

**Comment by philipp-spiess** [comment]
@taras-yemets Makes sense! I vividly remembered that we already shipped this so I got confused 😅

**Review by philipp-spiess — APPROVED** [review]
Let's do this!

File: client/web/src/integration/blob-viewer.test.ts [hunk 1]
```diff
@@ -44,6 +44,13 @@ describe('Blob viewer', () => {
 
     const commonBlobGraphQlResults: Partial<WebGraphQlOperations & SharedGraphQlOperations> = {
         ...commonWebGraphQlResults,
+        ...createViewerSettingsGraphQLOverride({
+            user: {
+                experimentalFeatures: {
+                    enableCodeMirrorFileView: false,
+                },
+            },
+        }),
```
Comment (philipp-spiess) [inline client/web/src/integration/blob-viewer.test.ts:53]: We should instead fix the tests to work with CodeMirror 😅

Comment (taras-yemets) [reply client/web/src/integration/blob-viewer.test.ts:53]: We have a [similar test](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@c87fd3552eb63e47378cf70bc5a1302040e9c14b/-/blob/client/web/src/integration/codemirror-blob-view.test.ts?subtree=true) for the CodeMirror blob viewer.

**Pull request merged**